### ISSUE TITLE
Infinite Loop on Reddit

### ISSUE THREAD

**Issue by JesseAldridge** [proginosko/LeechBlockNG-chrome#14]
The Leechblock page just keeps appearing over and over after the timer expires.
I recorded a video demonstrating the issue:
https://www.dropbox.com/s/poas4rx33tefd14/leechblock-loop.mov?dl=0

Reddit is the only site I've seen where this happens.
I just tested Hacker News and Twitter and Leechblock works properly there.

**Comment by JesseAldridge** [comment]
Here are my settings:

<img width="854" alt="settings1" src="https://user-images.githubusercontent.com/191903/71305681-55cb4580-238c-11ea-8c7d-b48b49f2f725.png">
<img width="866" alt="settings2" src="https://user-images.githubusercontent.com/191903/71305683-58c63600-238c-11ea-9cbe-9d5ddb632297.png">

**Comment by simha-r** [comment]
Yep, I have the same issue on reddit.com.

**Comment by JesseAldridge** [comment]
The fix was to switch to [inmotion.app](https://www.inmotion.app/)
(Sorry, proginosko)

**Comment by PietroProraso** [comment]
This extension would be great - if only it would work for reddit. 

Please fix this issue!

**Comment by proginosko** [comment]
I've looked into this, but I haven't been able to reproduce the issue on my system.

Some questions:

1. Which OS are y'all using?
2. Does the issue still occur when all other extensions are disabled?
3. Does it make any difference whether you enter `reddit.com` or `www.reddit.com` in the location bar?
4. Does it make any difference whether you're signed in/out of your reddit account?

Thanks in advance for any answers you can provided. It may help to trace the problem.

**Comment by PietroProraso** [comment]
1. running Debian buster (Linux 4.19.0-8-amd64 #1 SMP Debian 4.19.98-1 (2020-01-26) x86_64 GNU/Linux)
2. I disabled other extensions. Yes the problem still persists. For whatever reason, a few times whenI was testing just now, this problem did not appear. I think only when I accessed reddit via bookmarked links. Most of the time, the problem persists. 
3. both produce the same behavior. (I block with the string "*reddit.com")
4. nope
